src/nni_manager/training_service/reusable/aml/amlClient.ts

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+'use strict';
+
+import * as fs from 'fs';
+import * as request from 'request';
+import * as path from 'path';
+import { Deferred } from 'ts-deferred';
+import { PythonShell } from 'python-shell';
+
+export class AMLClient {
+    public subscriptionId: string;
+    public resourceGroup: string;
+    public workspaceName: string;
+    public experimentId: string;
+    public image: string;
+    public scriptName: string;
+    public pythonShellClient: undefined | PythonShell;
+    public codeDir: string;
+    public nodeCount: number;
+    public computeTarget: string;
+
+    constructor(
+        subscriptionId: string,
+        resourceGroup: string,
+        workspaceName: string,
+        experimentId: string,
+        computeTarget: string,
+        nodeCount: number,
+        image: string,
+        scriptName: string,
+        codeDir: string,
+        ) {
+        this.subscriptionId = subscriptionId;
+        this.resourceGroup = resourceGroup;
+        this.workspaceName = workspaceName;
+        this.experimentId = experimentId;
+        this.image = image;
+        this.nodeCount = nodeCount;
+        this.scriptName = scriptName;
+        this.codeDir = codeDir;
+        this.computeTarget = computeTarget;
+    }
+
+    public async submit(): Promise<string> {
+        const deferred: Deferred<string> = new Deferred<string>();
+        this.pythonShellClient = new PythonShell('amlUtil.py', {
+            scriptPath: './config/aml',
+            pythonOptions: ['-u'], // get print results in real-time
+            args: [
+                '--subscription_id', this.subscriptionId,
+                '--resource_group', this.resourceGroup,
+                '--workspace_name', this.workspaceName,
+                '--computer_target', this.computeTarget,
```

Comment:
better change this to `compute_target` for consistency

Reply:
fixed.